ui/src/app/applications/components/application-details/application-details.tsx

```diff
@@ -146,6 +146,18 @@ export class ApplicationDetails extends React.Component<RouteComponentProps<{nam
                                 return this.filterTreeNode(resNode, treeFilter);
                             });
 
+                            const URL_REGEX = /https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)/;
```

Comment:
Nitpick: We already have URL regex defined in application-summary.tsx:


https://github.com/argoproj/argo-cd/blob/master/ui/src/app/applications/components/application-summary/application-summary.tsx#L19

Can you please move it to https://github.com/argoproj/argo-cd/blob/master/ui/src/app/applications/components/utils.tsx and re-use in both application-summary.tsx and application-details.tsx

Reply:
sure